Fail to open the browser after startup on linux without Desktop.browse support

When starting OpenRefine on a Linux machine where the Desktop.browse API isn't supported, I get the following error:
```
09:53:13.377 [            refine_server] Java runtime version 17.0.11+9-Debian-1 from java.home: /usr/lib/jvm/java-17-openjdk-amd64 (0ms)
09:53:13.380 [            refine_server] Java VM: Debian OpenJDK 64-Bit Server VM 17.0.11+9-Debian-1 (2ms)
09:53:13.380 [            refine_server] Starting Server bound to '127.0.0.1:3333' (0ms)
09:53:13.380 [            refine_server] refine.memory size: 1400M JVM Max heap: 1400.0 MBytes (0ms)
09:53:13.409 [            refine_server] Initializing context: '/' from '/tmp/tmpclone/main/webapp' (29ms)
09:53:13.442 [            refine_server] Starting autoreloading scanner...  (32ms)
09:53:13.458 [..pse.jetty.server.Server] jetty-10.0.16; built: 2023-08-25T19:11:25.422Z; git: a2735a9ae9d1afc124974253f1e223e5678be1f4; jvm 17.0.11+9-Debian-1 (15ms)
09:53:13.786 [..dardDescriptorProcessor] NO JSP Support for /, did not find org.eclipse.jetty.jsp.JettyJspServlet (328ms)
09:53:13.792 [..DefaultSessionIdManager] Session workerName=node0 (5ms)
09:53:13.807 [...handler.ContextHandler] Started o.e.j.w.WebAppContext@c1fca1e{OpenRefine,/,file:///tmp/tmpclone/main/webapp/,AVAILABLE}{/tmp/tmpclone/main/webapp} (15ms)
09:53:13.826 [..erver.AbstractConnector] Started ServerConnector@6bd61f98{HTTP/1.1, (http/1.1)}{127.0.0.1:3333} (18ms)
09:53:13.834 [..pse.jetty.server.Server] Started RefineServer@6f36c2f0{STARTING}[10.0.16,sto=30000] @864ms (8ms)
Exception in thread "main" java.lang.NoClassDefFoundError: org/apache/commons/lang3/SystemUtils
	at com.google.refine.RefineClient.openBrowserFallback(Refine.java:514)
	at com.google.refine.RefineClient.openBrowser(Refine.java:498)
	at com.google.refine.RefineClient.init(Refine.java:484)
	at com.google.refine.Refine.init(Refine.java:139)
	at com.google.refine.Refine.main(Refine.java:116)
Caused by: java.lang.ClassNotFoundException: org.apache.commons.lang3.SystemUtils
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:525)
	... 5 more

```

This is due to us relying on the `SystemUtils` class to open the browser. This class belongs to the `commons-lang3` library which isn't available in the classpath of the `server` module.

Running `mvn dependency:analyze` indeed reports that this is a used but undeclared dependency of this module:
```
[INFO] --- maven-dependency-plugin:3.6.1:analyze (default-cli) @ server ---
[WARNING] Used undeclared dependencies found:
[WARNING]    org.apache.commons:commons-lang3:jar:3.14.0:provided
[WARNING]    org.apache.logging.log4j:log4j-api:jar:2.23.1:compile
[WARNING]    org.eclipse.jetty:jetty-util:jar:10.0.16:compile
```

--- a/main/pom.xml
+++ b/main/pom.xml
@@ -218,6 +218,7 @@
       <groupId>org.apache.commons</groupId>
       <artifactId>commons-lang3</artifactId>
       <version>${commons-lang3.version}</version>
+      <scope>provided</scope>
     </dependency>
     <dependency>
       <groupId>commons-fileupload</groupId>

--- a/server/pom.xml
+++ b/server/pom.xml
@@ -175,9 +175,19 @@
       <version>${jetty.version}</version>
     </dependency>
     <dependency>
+      <groupId>org.eclipse.jetty</groupId>
+      <artifactId>jetty-util</artifactId>
+      <version>${jetty.version}</version>
+    </dependency>
+    <dependency>
       <groupId>org.openrefine.dependencies</groupId>
       <artifactId>butterfly</artifactId>
       <version>${butterfly.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.logging.log4j</groupId>
+      <artifactId>log4j-api</artifactId>
+      <version>${log4j.version}</version>
     </dependency>
     <dependency>
       <groupId>org.apache.logging.log4j</groupId>
@@ -198,6 +208,11 @@
       <groupId>org.apache.logging.log4j</groupId>
       <artifactId>log4j-1.2-api</artifactId>
       <version>${log4j.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-lang3</artifactId>
+      <version>${commons-lang3.version}</version>
     </dependency>
   </dependencies>
 </project>

